resources/dmmd-preview.js

```diff
@@ -1,10 +1,20 @@
 $(function () {
     window.register_dmmd_preview = function ($preview) {
+
+        var $form = $preview.parents('form').first();
         var $update = $preview.find('.dmmd-preview-update');
         var $content = $preview.find('.dmmd-preview-content');
         var preview_url = $preview.attr('data-preview-url');
         var $textarea = $('#' + $preview.attr('data-textarea-id'));
 
+        // submit the form if ctrl + enter is pressed in pagedown textarea
+        $textarea.keydown(function (ev) {
+            // Ctrl-Enter pressed
+            if (ev.ctrlKey && ev.keyCode == 13) {
```

Comment:
This doesn't work, at least on my Chrome install. Pressing Ctrl+Enter into the textbox at https://api.jquery.com/keypress/ has the `keyCode` be 10, and just Enter be 13. `.keyCode` isn't very cross-browser-compatible; you should be using the jQuery-provided `.which` instead.